app/src/main/java/com/example/hungrywolfs/adapters/BindingAdapters.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.hungrywolfs.adapters
+
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.net.toUri
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.example.hungrywolfs.R
+import com.example.hungrywolfs.network.MealInfo
+
+@BindingAdapter("strMealThumb")
+fun bindCircleImage(imgView: ImageView, imgUrl: String?) {
+    imgUrl?.let {
+        val imgUri = imgUrl.toUri()
+        imgView.load(imgUri) {
+            placeholder(R.drawable.loading_animation)
+            error(R.drawable.ic_broken_image)
+        }
+    }
+}
+
+@BindingAdapter("strMeal")
+fun bindText(textView: TextView, text: String?) {
```

Comment:
An improvement for these 2 adapters is to remove the first parameter and actually create an [extension function](https://kotlinlang.org/docs/extensions.html#extension-functions) like you created for the RecyclerView below.